tests/utils/test_logging.py

```
@@ -554,35 +557,165 @@ def test_cuda_function(tensor):
+
+        @flashinfer_api
+        def _id(x):
+            return x
+
+        x = torch.zeros(8, device="cuda")
+
+        graph = torch.cuda.CUDAGraph()
+        try:
+            with torch.cuda.graph(graph):
+                _id(x)
+        except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)